.github/workflows/build_cmake.yaml

```
@@ -82,10 +84,25 @@
+
+      - name: Setup compiler cache for PRs
+        uses: hendrikmuhs/ccache-action@v1.2.18
+        if: ${{ github.event.number != '' }}
+        with:
+          variant: ${{ contains(matrix.cmake_preset, 'msvc') && 'sccache' || 'ccache' }}
+          key: ${{ matrix.cmake_preset }}-PR-${{ github.event.number }}
+          restore-keys: |
+            ${{ matrix.cmake_preset }}-PR-${{ github.event.number }}
+            ${{ matrix.cmake_preset }}-branch-${{ github.base_ref }}
+            ${{ matrix.cmake_preset }}-
+          create-symlink: ${{ !contains(matrix.contains, 'msvc') }}
```

actionlint (1.7.7)

property "contains" is not defined in object type {cmake_preset: string; continue_on_error: bool; cpack_extension: string; cpack_type: string; name: string; os: string; run_test: bool; run_test_with_results: bool}

(expression)